src/test/fuzz.cpp

```diff
@@ -88,9 +89,16 @@ tryRead(XDRInputFileStream& in, StellarMessage& m)
     }
 }
 
-#define PERSIST_MAX 1000
-static unsigned int persist_cnt = 0;
+void
+seedRandomness()
+{
+    srand(1);
+    gRandomEngine.seed(1);
+}
 
+#define PERSIST_MAX 1000
```

Comment:
This is the recommended numerical value from [AFL llvm mode docs](https://github.com/mirrorer/afl/tree/master/llvm_mode). 

> The number passed to __AFL_LOOP() controls the maximum number of
     iterations before the loop exits and the program is allowed to
     terminate normally. This limits the impact of accidental memory leaks
     and similar hiccups. 1000 is a good starting point, and going much higher
    increases the likelihood of hiccups without giving you any real performance benefits.

From what I have seen online (blogs, other projects, the docs) 1000 is a very commonly used magic number.

Reply:
OK, so it's actually not a recommended value: if you think about it, it's a way to amortize startup and setup cost
Reason I was asking about it is that it seems very low compared to our startup cost. If the goal is to avoid memory leaks, it sounds like 1000 is a setting maybe better suited for large payloads (like when people fuzz images). As a message is in the order of a few hundred bytes, I imagine that we can go a lot higher on this number (which should increase exploration rate without hurting stability).